central/scannerdefinitions/handler/handler.go

```diff
@@ -49,7 +49,9 @@ const (
 	// TODO(ROX-20481): Replace this URL with prod GCS bucket domain
 	v4StorageDomain   = "https://storage.googleapis.com/scanner-v4-test"
 	mappingFile       = "redhat-repository-mappings/mapping.zip"
+	cvssFile          = "nvd-bundle/nvd-data.tar.gz"
```

Comment:
have we implemented the CVSS enricher yet? Are we sure it will be one universal file instead of a version-specific one?

Reply:
CVSS should not be versioned because we get the json files from NVD and send it all the way to scanner